src/bundles.ts

```diff
@@ -183,15 +183,32 @@ interface CreateBundleFromFoldersOptions extends ConnectionOptions, AnalyzeFolde
  * @param source
  * @returns
  */
-async function getSupportedFiles(baseURL: string, source: string, requestId?: string): Promise<SupportedFiles> {
+async function getSupportedFiles(
+  baseURL: string,
+  source: string,
+  requestId?: string,
+  languages?: string[],
+): Promise<SupportedFiles> {
   emitter.supportedFilesLoaded(null);
   const resp = await getFilters(baseURL, source, undefined, requestId);
   if (resp.type === 'error') {
     throw resp.error;
   }
-  const supportedFiles = resp.value;
-  emitter.supportedFilesLoaded(supportedFiles);
-  return supportedFiles;
+  const supportedFilesFromApi = resp.value;
+  //Given supported languages from 'registy'
+  if (languages) {
```

Comment:
question: does this basically check the response (of supported languages) and verifies whether CLI options are not 💩 ?

Reply:
@pkey, What do you mean "by whether CLI options are not 💩 ?" 🤔

Reply:
it looks to me that the code checks languages coming from `options` which are CLI arguments right?

Reply:
Anyways, I approve not to block it.

Reply:
@pkey it is also does verification that all supportedLanguages including beta languages are being supported by the backed (/filters)